### PR TITLE
Format model

### PR DESCRIPTION
**Pull request by SanderGi**
## What did I do?
Updated the deprecatedEnumerateInstances API to instead use the FormattingModel API (to go with Visual API version 5.1+). Clarified some UI to make it more intuitive and expanded font size inputs to full font control inputs (font size, family, style, etc.).

## Why did I do it this way?
This will allow making changes using newer features and bugfixes from the powerbi-apis. One example, is the extended font control that this PR adds which allows users more control over how their visual looks. This PR enables some accessibility improvements in [this other PR](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95) which will make the sunburst chart usable for more users.

## Anything else you should know?
Translations for the new font control inputs have not been added since I don't speak all the required languages.

## Screenshots
Before:
image
After:
image

### CODE REVIEW

**Comment by zBritva** [comment]
image

I guess the name of property "Show category label" is confusing. It's font size of Selected category label. 
I this is better to rename to "Show selected category label".

@MulyukovAidar what do you think?

**Comment by zBritva** [comment]
@MulyukovAidar do you extend fond properties when you migrate to new formatting model? (Intriduce font family, font type, size for exsisting labels)

image

**Comment by SanderGi** [comment]
@zBritva Both the "Show Category" property name and the font controls have been updated: 
image

**Comment by SanderGi** [comment]
Added better accessibility to the visual based on the new slices using correct aria-attributes:
image

**Comment by SanderGi** [comment]
The accessibility features are now their own PR: https://github.com/microsoft/powerbi-visuals-sunburst/pull/95

**Comment by zBritva** [comment]
image

Font control doesn't applies font family and font type. Font size works.

**Comment by SanderGi** [comment]
> Font control doesn't applies font family and font type instantly. Font size works. User need to reselect the category to apply new font proiperties

Hmm, I don't see this behavior. Can you link the dataset you're using?

Recording 2023-07-24 at 10 14 37

**Comment by FirzinatKhuzeev** [comment]
> > Font control doesn't applies font family and font type instantly. Font size works. User need to reselect the category to apply new font proiperties
> 
> Hmm, I don't see this behavior. Can you link the dataset you're using?
> 
> Recording 2023-07-24 at 10 14 37

Can you please add gif with visualization pane? (to see your changes)

**Comment by SanderGi** [comment]
> > > Font control doesn't applies font family and font type instantly. Font size works. User need to reselect the category to apply new font proiperties
> > 
> > 
> > Hmm, I don't see this behavior. Can you link the dataset you're using?
> > Recording 2023-07-24 at 10 14 37
> >       
> >         [
> >           
> >         
> >             Recording 2023-07-24 at 10 14 37
> >           ](https://user-images.githubusercontent.com/97496861/255665337-570509af-ad9d-4d61-bd53-60f39876dd40.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/97496861/255665337-570509af-ad9d-4d61-bd53-60f39876dd40.gif)
> 
> Can you please add gif with visualization pane? (to see your changes)

Here's the gif: Recording 2023-07-26 at 13 42 34

File: capabilities.json [hunk 1]
```diff
@@ -42,35 +42,76 @@
   ],
   "objects": {
     "group": {
-      "displayName": "Group",
-      "displayNameKey": "Visual_Group",
       "properties": {
         "showSelected": {
-          "displayName": "Show category label",
-          "displayNameKey": "Visual_ShowCategoryLabel",
```
Comment (uve) [inline capabilities.json:50]: Do not remove displayNameKey strings

**Review by uve — CHANGES_REQUESTED** [review]

File: capabilities.json (hunk 1 above)
Comment (SanderGi) [reply capabilities.json:50]: See https://github.com/microsoft/powerbi-visuals-sunburst/pull/95#discussion_r1283043020. We do not need the keys here.

**Comment by uve** [comment]
@SanderGi could you please merge #94 and #95 PRs to make a single working PR

**Comment by SanderGi** [comment]
> @SanderGi could you please merge #94 and #95 PRs to make a single working PR

Will do. Closing this PR, #95 will be the new merged PR

**Pull request closed without merging**